core/templates/dev/head/pages/exploration_editor/simple_editor_tab/questions/MultipleChoiceEditorDirective.js

```diff
@@ -101,7 +103,10 @@ oppia.directive('multipleChoiceEditor', [
             }
 
             if (foundEmptyField) {
-              return;
+              return StatusObjectFactory.createNew(
```

Comment:
@seanlip I'm starting to think this code is a bit too much logic and should probably go into a service so it can be easily tested. We should have tests verifying the correct status is returned given the correct conditions, otherwise the directive is getting a bit too heavy.